docs/noise.ipynb

```diff
@@ -218,130 +218,6 @@
     "Cirq provides the helpers `cirq.generalized_amplitude_damp` and\n",
     "`cirq.amplitude_damp` to construct these noisy gates."
    ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "id": "b3625a58b817"
-   },
-   "source": [
-    "### GeneralizedAmplitudeDampingChannel and AmplitudeDampingChannel\n",
-    "\n",
-    "The generalized amplitude damping channel models the effect of energy\n",
-    "dissipation to a surrounding environment as well as dephasing that\n",
-    "does not exchange energy. The amplitude damping channel only models dissipation of energy to a surrounding environment. \n",
-    "\n",
-    "Cirq has implementations of both of these channels. The generalized amplitude damping channel corresponds to:\n",
-    "\n",
-    "$$\n",
-    "\\begin{aligned}\n",
-    "\\rho \\rightarrow& \\sum_{k=0}^3 M_k \\rho M_k \\newline\n",
-    "M_0 =& \\sqrt{p} \\begin{bmatrix} 1 & 0  \\cr 0 & \\sqrt{1 - \\gamma} \\end{bmatrix} \\newline\n",
-    "M_1 =& \\sqrt{p} \\begin{bmatrix} 0 & \\sqrt{\\gamma} \\cr 0 & 0 \\end{bmatrix} \\newline\n",
-    "M_2 =& \\sqrt{1-p} \\begin{bmatrix} \\sqrt{1-\\gamma} & 0 \\cr 0 & 1 \\\\ \\end{bmatrix} \\newline\n",
-    "M_3 =& \\sqrt{1-p} \\begin{bmatrix} 0 & 0 \\cr \\sqrt{\\gamma} & 0 \\end{bmatrix}\n",
-    "\\end{aligned}\n",
-    "$$\n",
-    "\n",
-    "Where &#947; is the probability of the interaction being dissipative, and\n",
-    "`p` is the probability that the qubit and environment exchange energy. The amplitude damping channel corresponds to `p=1`.\n",
-    "\n",
-    "Cirq provides the helpers `cirq.generalized_amplitude_damp` and\n",
-    "`cirq.amplitude_damp` to construct these noisy gates."
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {
-    "id": "f3933141f64e"
-   },
-   "source": [
-    "### cirq.mixture and def _mixture_\n",
```

Comment:
Optional: I think underscores will turn this into italics instead of displaying as `_mixture_`, so there should be backticks around `"def _mixture_"`.

While you're at it, consider fixing in the copy which you're keeping.

Reply:
Done.